ports/rp2/mphalport.c

```diff
@@ -44,6 +44,30 @@ ringbuf_t stdin_ringbuf = { stdin_ringbuf_array, sizeof(stdin_ringbuf_array) };
 
 #endif
 
+#if MICROPY_HW_ENABLE_USBDEV
+
+#ifndef USB_BUFFER_LEN
+#define USB_BUFFER_LEN 512
+#endif
+
+
+STATIC uint8_t usb_ringbuf_array[USB_BUFFER_LEN];
+ringbuf_t usb_ringbuf = { usb_ringbuf_array, sizeof(usb_ringbuf_array) };
```

Comment:
I think it would make sense to reused the uart ringbuffer.  So just make it a common (always defined) ringbuf called `stdin_ringbuf` (like mimxrt does it).